README.md

```diff
@@ -1,12 +1,12 @@
-# inform7-bugs
-This is an unofficial bug tracker for Inform 7. 
+# inform7-bugs: now retired
 
-Please post interface bugs in the Inform 7 IDEs at these places instead:
+This repo provided an unofficial bug tracker for Inform 7 6M62. There is once again an [official Inform7 bug tracker](https://inform7.atlassian.net/jira/software/c/projects/I7/issues), so there's no further need to report bugs in this project.
+
+Please post interface bugs in the Inform 7 IDEs at these places:
```

Comment:
It's my understanding that the IDEs will use the official bug tracker as well — I'm not 100% sure though!